test/create.spec.js

```diff
@@ -29,4 +29,22 @@ describe('dsFirebaseAdapter#create', function () {
         assert.deepEqual(user, { id: id, name: 'John' });
       });
   });
+
+  it('should create a user in firebase, created users primary key should have no leading slash', function () {
+
+    UserDiffId = store.defineResource({
```

Comment:
Don't define a new resource for this test, just re-use the User resource.

As it is written, every time this test runs, it will add data to firebase, but that data won't be cleaned up when the tests are done.